### ISSUE TITLE
Little Inferno Lacks Cursor and Fire doesn't work.

### ISSUE THREAD

**Issue by Fancy2209** [cemu-project/Cemu#1056]
[log.txt](https://github.com/cemu-project/Cemu/files/13802297/log.txt)

https://github.com/cemu-project/Cemu/assets/64917206/49e32fdc-488f-4b92-8267-514fc67decd2

When using a Wii Remote the Cursor does not show up (works in the WiiU Menu), and with touch the fire doesn't work.

**Comment by Squall-Leonhart** [comment]
The cursor looks to be the same one provided by ErrEula, it may require CafeLibs and corresponding title files.

**Comment by Fancy2209** [comment]
> The cursor looks to be the same one provided by WiiU for the system menu, it may require CafeLibs and corresponding title files, however IR is very spotty on Cemu currently so it may just not be pointing onscreen ata ll.

I do have the cafeLibs

**Comment by Squall-Leonhart** [comment]
Not all of them.

**Comment by Fancy2209** [comment]
> Not all of them.

Which one would I need for the cursor? Probably failed to copy it

**Comment by Squall-Leonhart** [comment]
No, i mean Cemu only allows the use of a few of the total set of CafeLib files and the cursor could be supplied by one of the ones it currently doesn't and can't use, I don't know which in particular that might be presently.

**Comment by Fancy2209** [comment]
Ahhh, okay I see, my bad

**Comment by Fancy2209** [comment]
> @Fancy2209 Could you try #1127 in regards to hopefully fixing the IR pointer

I'm so sorry, I forgot about this
I'll test it this week
